x-pack/plugins/stack_alerts/public/alert_types/es_query/expression/search_source_expression.scss

```diff
@@ -1,3 +1,7 @@
+html {
+  overflow: hidden;
```

Comment:
Instead, is it possible to set the `suggestionSize` on the component to `s`?

I see a prop here: https://github.com/elastic/kibana/blob/3b37b27826fb76e12cef5b6091ef4d738a51bbe5/src/plugins/unified_search/public/typeahead/suggestions_component.tsx#L160

I am not sure where/how to pass in this prop, so (as a very quick-and-dirty test) I hardcoded this value to `s` and get this result which I suspect addresses the original issue:

<img width="628" alt="Screen Shot 2022-04-20 at 4 30 28 PM" src="https://user-images.githubusercontent.com/446285/164327204-6fc913e6-7730-485f-976b-996f4070977c.png">

Reply:
Yes, it's possible, already applied.